static/js/pm_list.js

```diff
@@ -123,6 +124,25 @@ function unhighlight_all_private_messages_view() {
     $(".private_messages_container").removeClass("active_private_messages_section");
 }
 
+function get_user_id() {
+    const current_id = pm_list_data.get_active_user_ids_string();
+    return current_id;
+}
+
+export function get_user_li() {
+    const $current_id = get_user_id();
```

Comment:
A dollar sign in a variable name is reserved for a `jQuery` object -- and thus ID is an integer.

Also I think the `get_user_id` helper obscures things; removed it in favor of directly accessing `pm_list_data.get_active_user_ids_string`, which says more precisely what this value is.